constance/backends/redisd.py

```diff
@@ -35,7 +36,7 @@ def add_prefix(self, key):
     def get(self, key):
         value = self._rd.get(self.add_prefix(key))
         if value:
-            return loads(value)
+            return loads(value)  # noqa: S301
```

Comment:
That's exactly why I don't like using Pickle for that...
1. It's not portable in between Python version
2. It's not portable in between languages, hard to read when you look at your redis, etc.

Reply:
Faced this in my project when we need read constance value in another service from db. But service is that written in another language. This should be rethinked

Reply:
As an idea. 
I think it's possible to use modified `django.db.models.JSONField`.
And saving something like 
```
{
	"type": "int",
	"value": "1",
}
```
And casting it to the right value in to_reprenetaation

Reply:
This merge request is dedicated to linters so let`s leave it.
I agree that pickle is not the best choice.